caas/kubernetes/provider/resources/applier.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package resources
+
+import (
+	"context"
+
+	"github.com/juju/errors"
+	"github.com/juju/loggo"
+	"k8s.io/client-go/kubernetes"
+)
+
+var logger = loggo.GetLogger("juju.kubernetes.provider.resources")
+
+type applier struct {
+	ops []operation
+}
+
+// NewApplier creates a new applier.
+func NewApplier() Applier {
```

Comment:
I wonder if "Applier" could be ClusterOperation so it is named like our ModelOperation interface